### PR TITLE
HAWQ-885. Should fail earlier at creating for a table with quicklz co…

### PR DESCRIPTION
**Pull request by paul-guo-**
…mpression.

### CODE REVIEW

**Comment by wengyanqing** [comment]
LGTM

**Comment by ztao1987** [comment]
+1

File: src/backend/access/common/reloptions.c [hunk 1]
```diff
@@ -515,22 +515,20 @@ default_reloptions(Datum reloptions, bool validate, char relkind,
 
 		compresstype = values[5];
```
Comment (huor) [inline src/backend/access/common/reloptions.c]: The fix should remove quicklz from valid compression type list. It is better to add comment here we reserve quicklz in catalog for ease of upgrade.

Comment (paul-guo-) [inline src/backend/access/common/reloptions.c]: The code is not just for hacking here. Even we remove quicklz from system tables in the future, this code could be still kept to educate users for some time.

**Pull request merged**